src/test/java/com/band/api/MutationTest.java

```diff
@@ -0,0 +1,70 @@
+package com.band.api;
+
+import com.band.api.Exceptions.CustomException;
+import com.band.api.domain.User;
+import com.band.api.repository.UserRepository;
+import com.band.api.resolvers.Mutation;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mockito;
+import org.springframework.dao.DataAccessResourceFailureException;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+
+class MutationTest {
+    static UserRepository mockUserRepository;
+    static PasswordEncoder mockPasswordEncoder;
+    static Mutation mutation;
+
+    @Test
+    @BeforeAll
```

Comment:
@ Before instead of @ BeforeAll, will make it much easier to update the tests in the future and ensures we don't have tests causing side effects for each other. If performance becomes an issue we can update this then.